bin/inference/pycbc_inference_plot_samples

```diff
@@ -44,44 +52,87 @@ opts = parser.parse_args()
 # setup log
 pycbc.init_logging(opts.verbose)
 
+rc('text', usetex=True)
+
```

Comment:
Delete this. I don't like the font you get with `usetex` set to True; the standard font is nicer.